docs/schema/content-migration-schemas.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Content migration schemas
+description: Describes the Content migration schema files for SharePoint.
 manager: soliver
 ms.date: 3/9/2015
```

Comment:
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.